### ISSUE TITLE
Warnings for tap_raw:diag/2 and tap_raw:is_test_running/0

### ISSUE THREAD

**Issue by meditans** [nicoabie/quickcheck#12]
Steps to reproduce the problem:
1) install the quickcheck library
2) open swipl
3) use_module(library(quickcheck)).
4) make.

At this point, I get the following warnings:
```
Warning: The predicates below are not defined. If these are defined
Warning: at runtime using assert/1, use :- dynamic Name/Arity.
Warning: 
Warning: tap_raw:diag/2, which is referenced by
Warning: 	/home/carlo/lib/swipl/pack/quickcheck/prolog/quickcheck.pl:135:41: 1-st clause of quickcheck:warn/2
Warning: tap_raw:is_test_running/0, which is referenced by
Warning: 	/home/carlo/lib/swipl/pack/quickcheck/prolog/quickcheck.pl:135:14: 1-st clause of quickcheck:warn/2
```

I can't explain why these errors happen, as `tap_raw:is_test_running/0` is indeed marked as dynamic in the last version of `tap`, and `tap_raw:diag/2` is defined. I'd like to know if you can reproduce this behaviour, and if that's not the case, any insight on what I'm doing wrong (my goal is to use mavis without warning at every make command).

**Comment by mndrix** [comment]
Thanks for the report.  I'm no longer maintaining this repository and don't have a set up where I can test it.  Sorry that I can't be more helpful.

**Comment by nicoabie** [comment]
Fixed in version v0.2.1
